2024-HPCIC-AWS/JupyterNotebook/aws/config-aws-ssl.yaml

```diff
@@ -26,6 +26,15 @@ hub:
     tag: "hpcic-2024"
     pullPolicy: Always
 
+  # Increase the resources for large tutorials
+  resources:
```

Comment:
This won't lead to setting limits/requests and the pod in Guaranteed and OOM errors and the notebooks being killed?

Reply:
I updated it. In general if you have a Guaranteed pod with resource consumption very close to the physical limits of the machine it is likely to get OOM killed regardless of whether it's Guaranteed or not.